src/protocols/aeternity/helpers/transactionTokenInfoResolvers.ts

```diff
@@ -257,21 +263,23 @@ const swapExactTokensForAe = genLiquiditySwapResolver(
     amount: _arguments[0]?.value, // amount_a_desired: int
   }),
   ({ tx: { arguments: _arguments } }) => ({
-    contractId: _arguments[2]?.value?.[_arguments[2]?.value?.length - 1]?.value,
+    contractId: AE_CONTRACT_ID,
     maxAmount: _arguments[1]?.value,
     amount: _arguments[1]?.value, // amount_b_desired: int
-    isAe: true,
+    name: AE_COIN_NAME,
+    symbol: AE_SYMBOL,
   }),
 );
 
 /**
  * ref: AedexV2Router.swap_ae_for_exact_tokens
  */
 const swapAeForExactTokens = genLiquiditySwapResolver(
-  ({ tx: { arguments: _arguments, amount } }) => ({
-    contractId: _arguments[1]?.value?.[0]?.value,
+  ({ tx: { amount } }) => ({
+    contractId: AE_CONTRACT_ID,
```

Comment:
Why don't you use `...defaultCoin,` here instead?

Reply:
Right, this was a leftover from my first approach. Updated...